beamer/state_machine.py

```diff
@@ -219,17 +209,14 @@ def _handle_claim_made(event: ClaimMade, context: Context) -> HandlerResult:
     except TransitionNotAllowed:
         return False, events
 
-    if request.l1_resolution_tx_handle is None and _l1_resolution_criteria_fulfilled(
-        claim, context
-    ):
-        request.l1_resolution_tx_handle = context.resolution_pool.submit(
-            run_relayer,
-            context.config.l1_rpc_url,
-            context.config.l2b_rpc_url,
-            context.config.account.privateKey,
-            request.fill_tx,
-        )
-        log.info("Initiated L1 resolution", request=request, claim_id=event.claim_id)
+    if request.l1_resolution_tx_handle is None and request.fill_tx is not None:
+        events = [
+            InitiateL1ResolutionEvent(
+                chain_id=request.target_chain_id,  # Resolution happens on the target chain
+                request_id=request.id,
+                claim_id=claim.id,
+            )
+        ]
```

Comment:
I'm wondering, what will happen if we ignore this claim and then encounter this even in the next iteration?